composer.json

```diff
@@ -92,5 +171,27 @@
         "laravel-57-test": "composer --working-dir=tests/Frameworks/Laravel/Version_5_7 test",
         "symfony-34-update": "php -d memory_limit=-1 /usr/local/bin/composer --working-dir=tests/Frameworks/Symfony/Version_3_4 update",
         "symfony-34-test": "composer --working-dir=tests/Frameworks/Symfony/Version_3_4 test"
+    },
+    "extra": {
+        "scenarios": {
+            "elasticsearch1": {
+                "require": {
+                    "elasticsearch/elasticsearch": "1.2.*"
+                }
+            },
+            "guzzle5": {
+                "require": {
+                    "guzzlehttp/guzzle": "~5.0"
+                }
+            },
+            "predis1": {
+                "require": {
+                    "predis/predis": "^1.1"
+                }
+            }
+        },
+        "scenario-options": {
+            "dependency-licenses": false
```

Comment:
Maybe this should be the default. I like keeping my dependency licenses listed and up-to-date in the LICENSE file, but this feature is orthogonal to the primary purpose of composer-test-scenarios.

Reply:
> Maybe this should be the default.

It probably would make sense, as you suggested this is orthogonal to the purpose of this library. Very nice feature though, and the fact that it is on/off configurable denoted a very forward-thinking approach!